src/main/java/epicsquid/roots/entity/ritual/EntityRitualFlowerGrowth.java

```diff
@@ -75,8 +73,7 @@ private boolean generateFlower(BlockPos pos) {
       }
       IBlockState blockState = world.getBlockState(pos.down());
       Block block = blockState.getBlock();
-
-      IItemStack soil = new MCItemStack(new ItemStack(block, block.getMetaFromState(blockState)));
+      IItemStack soil = new MCItemStack(new ItemStack(block, 1, block.getMetaFromState(blockState)));
```

Comment:
Re: block, `ItemUtil::stackFromState` might simplify this down slightly. (It's in MysticalLib)